.github/workflows/ci.yml

```diff
@@ -9,13 +9,19 @@ jobs:
       - name: Check out repository
         uses: actions/checkout@v2
 
+      - name: Install pnpm
```

Comment:
I'm new to GH actions, but I'm curious: can we cache the `pnpm` installation between tasks? Or is that `with cache: "pnpm"` during the "Set up Node" step covering us?

Reply:
Ah gotcha, thanks for sharing those resources! Glad we're seeing a real performance improvement there 👍